packages/README.md

```diff
@@ -32,10 +33,39 @@ When creating a new package, you need to provide at least the following:
     	},
     	"publishConfig": {
     		"access": "public"
-    	}
+    	},
+    	// Include this line to include the package as a WordPress script.
+    	"wpScript": true,
+    	// Include this line to include the package as a WordPress script module.
+    	"wpScriptModuleExports": "./build-module/index.js"
```

Comment:
I'm not sure I like this. Why should a package.json mention a built file. why `wpModule: true` is not enough? since we already have `module` defined in the package.json ?

Reply:
I think one aspect needs to be documented. Using `wpScript` or/and `wpScriptModuleExports` marks the package as production package for strict verification of compatibility with GPL license.

`module` isn't configured correctly at the moment. Eventually, we can explore migrating to `"type": "module"` and `exports` configuration but it's a separate effort.

Reply:
> module isn't configured correctly at the moment.

I'm not sure I understand, we've been relying on module to output ESmodules forever, why is it not correct?

Reply:
I was looking at https://github.com/WordPress/gutenberg/blob/trunk/packages/a11y/package.json  for instance

That package has both "module" and "wpScriptModuleExports" and they point to different modules/APIs. So we're saying that this package ships "two different ESmodules" in one package. As an npm package consumer, I'm really confused what `import something from `@wordpress/a11y` will yield to me. Which one of these is going to be used? Ultimately a package can ship multiple modules, that's ok but not two versions of the same module.

Reply:
> I think one aspect needs to be documented. Using `wpScript` or/and `wpScriptModuleExports` marks the package as production package for strict verification of compatibility with GPL license.

I opened a follow-up https://github.com/WordPress/gutenberg/pull/66562.